### ISSUE TITLE
TS7006: Parameter 'error' implicitly has an 'any' type.

### ISSUE THREAD

**Issue by JamesCobbledick** [spatie/laravel-medialibrary-pro-app#8]
### Expected
With the command "npm run dev", the Vue components should compile and the upload form displayed.

### Actual
When running "npm run dev" I receive the following errors:

F:\localserver\htdocs\laravel-medialibrary-pro-app>npm run dev

> @ dev F:\localserver\htdocs\laravel-medialibrary-pro-app
> npm run development

> @ development F:\localserver\htdocs\laravel-medialibrary-pro-app
> mix

99% done plugins FriendlyErrorsWebpackPlugin

 ERROR  Failed to compile with 2 errors                                                                                                                                                                                         10:56:14

 error  in ./vendor/spatie/laravel-medialibrary-pro/resources/js/media-library-pro-vue3-attachment/dist/index.js

Module not found: Error: Can't resolve '@spatie/media-library-pro-vue3' in 'F:\localserver\htdocs\laravel-medialibrary-pro-app\vendor\spatie\laravel-medialibrary-pro\resources\js\media-library-pro-vue3-attachment\dist'

 error  in F:\localserver\htdocs\laravel-medialibrary-pro-app\resources\js\react\CollectionCustomProperty.tsx

[tsl] ERROR in F:\localserver\htdocs\laravel-medialibrary-pro-app\resources\js\react\CollectionCustomProperty.tsx(67,90)
      TS7006: Parameter 'error' implicitly has an 'any' type.

99% done plugins watchInfo[webpack-cli] Compilation finished

### Other issues
It's also worth noting, the vue upload forms don't display, but the react one does and when uploading to the blade component, the image thumbnail is a 404 and the console puts out the following:

Uncaught TypeError: Cannot set property 'value' of undefined
    at HTMLInputElement.clearFileInputValue (FileUploads.js:37)
    at eval (eval at <anonymous> (alpine.js:140), <anonymous>:3:33)
    at alpine.js:140
    at fe.evaluateCommandExpression (alpine.js:1679)
    at C (alpine.js:878)
    at HTMLDivElement.c (alpine.js:854)

**Comment by JamesCobbledick** [comment]
Also worth noting, this is the sample project that I've downloaded - not a fresh install, the webpack.mix.js is as follows:

```
const mix = require('laravel-mix');
const path = require('path');

mix.js('resources/js/vue/app.js', 'public/js/vue')
    .vue({ version: 3 })
    .ts('resources/js/react/app.tsx', 'public/js/react')
    .react()
    .js('resources/js/vapor/app.js', 'public/js/vapor');

mix.override((webpackConfig) => {
    delete webpackConfig.watchOptions;

    webpackConfig.resolve.symlinks = false;

    webpackConfig.resolve.alias = {
        ...webpackConfig.resolve.alias,
        react: path.resolve('./node_modules/react'),
        vue: path.resolve('./node_modules/vue'),
    };

    webpackConfig.resolve.modules = [`${__dirname}/vendor/spatie/laravel-medialibrary-pro/resources/js`, 'node_modules'];

    webpackConfig.resolve.extensions.push('.ts', '.tsx');
});

if (mix.inProduction()) {
    const ASSET_URL = process.env.ASSET_URL + '/';

    mix.webpackConfig((webpack) => {
        return {
            plugins: [
                new webpack.DefinePlugin({
                    'process.env.ASSET_PATH': JSON.stringify(ASSET_URL),
                }),
            ],
            output: {
                publicPath: ASSET_URL,
            },
        };
    });
} else {
    mix.postCss('resources/css/main.css', 'public/css', [require('tailwindcss')]);
}

```

**Comment by rino7** [comment]
Quick fix that works for me but probably was a workaround:

Set ` "noImplicitAny": false,` in this line
https://github.com/spatie/laravel-medialibrary-pro-app/blob/130731ddf80a58eb73915bfc69e53b39978e12be/tsconfig.json#L27

And then re-run npm run dev

**Comment by AdrianMrn** [comment]
The error you mentioned is just a type-error, and shouldn't stop the app from compiling. For some reason, TypeScript can't properly derive the type from this method: https://github.com/spatie/laravel-medialibrary-pro/blob/17e58092214e6af2d5572ba11806e397a38f4ff8/resources/js/media-library-pro-react-collection/dist/MediaLibraryCollection.d.ts#L35 I've fixed the ts error in my latest commit.

The composer dependencies have also been updated and your issue with the Vue components should also be fixed.
